assets/sidecar/developer-hub-app-config-patches.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+data:
+  RHDH_TOKEN: base64-encoded-token-generated-from-openssl-or-similar-program
```

Comment:
note ^^ that `RHDH_TOKEN` would need to be set with an actual encoded value for a legitimately built token